manifest/kcp.yaml

```diff
@@ -148,6 +148,13 @@ spec:
             path: readyz
             port: 6443
             scheme: HTTPS
+        resources:
+          limits:
+            cpu: '1'
+            memory: 1Gi
+          requests:
+            cpu: 500m
+            memory: 512Mi
```

Comment:
Do you want to start this high, given that we've observed the pod at ~120mb steady state (unless I misread the data)?